renderer/utils/reports.ts

```diff
@@ -216,3 +216,65 @@ export function validation(activities: Array<ReportActivity>) {
   }
   return activities;
 }
+
+export function addingSuggestions(
+  activities: Array<ReportActivity> | null,
+  latestProjAndDesc: Record<string, [string]>,
+  latestProjAndAct: Record<string, [string]>
+) {
+  for (let i = 0; i < activities.length; i++) {
+    if (!activities[i].project || activities[i].project?.startsWith("!")) {
+      continue;
+    }
+    const projectKey = activities[i].project.trim();
+
+    if (
+      Object.keys(latestProjAndDesc).length &&
+      !latestProjAndDesc.hasOwnProperty(projectKey) &&
+      activities[i].description &&
+      !latestProjAndAct.hasOwnProperty(projectKey)
+    ) {
+      latestProjAndDesc[projectKey] = [activities[i].description];
+      latestProjAndAct[projectKey] = [activities[i].activity];
+      continue;
+    }
+    if (
```

Comment:
can you make gaps between functions, it would make code more readable. In some places you have gaps in some you don't